src/bin/cargo-flamegraph.rs

```diff
@@ -306,11 +306,47 @@ fn workload(
     result
 }
 
+fn find_unique_bin_target() -> String {
```

Comment:
Let's also inline this single-use function.

Reply:
Sure I can inline them but it is quite long. That's why I split it off into another function to keep it more readable. Because of the function name, you can immediately see what it does which is not be the case if I'll inline the block (unless I add a describing comment).
The same goes for the other single use functions.

Reply:
I inlined all the other functions you suggested but I didn't inline this one yet. If you insist, I can inline it but I think `main` will be too long then.